.evergreen/config.yml

```diff
@@ -88,8 +88,9 @@ functions:
         script: >
           ${PREPARE_SHELL}
 
-          AUTH=${AUTH} SSL=${SSL} UNIFIED=${UNIFIED} MONGODB_URI="${MONGODB_URI}" bash
-          ${PROJECT_DIRECTORY}/.evergreen/run-tests.sh
+          AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
```

Comment:
Does this risk echoing out the AWS keys?

Reply:
yep, nice catch, fixed